actors/migration/nv16/top.go

```diff
@@ -289,17 +310,33 @@ func (job *migrationJob) run(ctx context.Context, store cbor.IpldStore, priorEpo
 }
 
 // Migrator which preserves the head CID and provides a fixed result code CID.
-type nilMigrator struct {
+type codeMigrator struct {
 	OutCodeCID cid.Cid
 }
 
-func (n nilMigrator) migrateState(_ context.Context, _ cbor.IpldStore, in actorMigrationInput) (*actorMigrationResult, error) {
+func (n codeMigrator) migrateState(_ context.Context, _ cbor.IpldStore, in actorMigrationInput) (*actorMigrationResult, error) {
 	return &actorMigrationResult{
 		newCodeCID: n.OutCodeCID,
 		newHead:    in.head,
 	}, nil
 }
 
-func (n nilMigrator) migratedCodeCID() cid.Cid {
-	return n.OutCodeCID
+// System Actor migrator
```

Comment:
Nit: the historic pattern has been to put this in its own file (eg https://github.com/filecoin-project/specs-actors/blob/release/v7/actors/migration/nv15/verifreg.go)

Reply:
ok, will do.

Reply:
done.